### DISCUSSION TITLE
Installing gulp results in "Failed at the node-sass@3.13.1 postinstall script."

### DISCUSSION THREAD

**Discussion by dylanh724** [gulpjs/gulp#2454] [Help]
Before you open this issue, please complete the following tasks:

### What were you expecting to happen?
Grub installs without errors

### What actually happened?
Error -- cannot install:
![image](https://user-images.githubusercontent.com/8840024/84366450-6b1b8100-ac05-11ea-843a-195ce2076bae.png)

### Please provide the following information:
* node version (run `node -v`): **v12.18.0**
* npm version (run `npm -v`): **6.14.5**
* gulp version (run `gulp -v`): **Not installed (trying!)**

### Additional Info
I had gulp before. I updated npm/node, then suddenly gulp broke. I tried to update it now have this issue.

**Comment by phated** [comment]
Gulp doesn't depend on node-sass, it sounds like you have a different issue with your install.

I'll convert this to a help Discussion in case another community member is able to help you look into that other issue.
